src/containers/AdminPersonList.jsx

```diff
@@ -43,13 +34,23 @@ class AdminPersonList extends React.Component {
     userEdit: false
   }
 
-  handleFilterChange = (event, index, value) => {
-    const query = value ? `?campaignId=${value}` : ''
+  handleFilterChange = (campaignId, offset) => {
+    let query = '?' + (campaignId ? `campaignId=${campaignId}` : '')
+    query += (offset ? `&offset=${offset}` : '')
     this.props.router.push(
       `/admin/${this.props.params.organizationId}/people${query}`
     )
   }
 
+  handleCampaignChange = (event, index, value) => {
```

Comment:
what are we using event and index for here? is this setting up for something else in the future?

Reply:
same question for `handleCampaignChange` and `handleOffsetChange`

Reply:
It seems to just be what arguments are sent on a DropDown change event -- I am just copying the old handleFilterChange method (on the left :-)
Before that was handling the only filterchange -- the champaign.  I changed that to be a more abstracted method that does the redirect, and then we now have two methods that handle different formchange events.

This is all kinda hacky -- the 'right way' would be that we'd add a state and refresh a query value, but apollo/graphql makes that so annoying that whoever added the campaign query did this instead which feels reasonable given how few people are on this page.